lib/tasks_private/spec.rake

```diff
@@ -4,8 +4,4 @@ namespace :spec do
 end
 
 desc "Run all specs"
-RSpec::Core::RakeTask.new(:spec => ["app:test:initialize", "app:evm:compile_sti_loader"]) do |t|
-  spec_dir = File.expand_path("../../spec", __dir__)
-  EvmTestHelper.init_rspec_task(t, ['--require', File.join(spec_dir, 'spec_helper')])
```

Comment:
This line wasn't supposed to be removed.  With this deletion we no longer use `.rspec_ci`